node/src/components/linear_chain_sync/operations.rs

```diff
@@ -128,12 +195,42 @@ where
         {
             Ok(FetchedData::FromStorage { item }) => return Ok(Some(item)),
             Ok(FetchedData::FromPeer { item, .. }) => {
-                // TODO: Validate item, ban peer if necessary
                 // Compute the total weight of the validators
```

Comment:
```suggestion
```

Reply:
Fixed in 01b0c705bac9f18e3f39f1adba231340f5c3e0b5